### ISSUE TITLE
how can I try packages that are already updated to 0.14 with spago

### ISSUE THREAD

**Issue by srghma** [purescript/package-sets#804]
seems like I have to:
- download this repo
- change version to "master" in files, e.g. in https://github.com/purescript/package-sets/blob/master/src/groups/purescript.dhall
- change

```
let upstream =
      https://github.com/purescript/package-sets/releases/download/psc-0.13.6-20200309/packages.dhall sha256:9221987b4e7ea99ccd0efbe056f7bebc872cd92e0058efe5baa181d73359e7b3
```

to 

````
let upstream =
      ~/packages/xxxx/src/packages.dhall
```

**Comment by srghma** [comment]
there is also https://github.com/purescript/package-sets/blob/prepare-0.14/src/groups/purescript.dhall

but it seems like outdated

am I missing some other way?

**Comment by JordanMartinez** [comment]
You should be using this file: https://github.com/purescript/package-sets/blob/prepare-0.14/src/packages.dhall

For example:
```dhall
{- spago.dhall -}
{ name = "library"
, packages = https://github.com/purescript/package-sets/blob/prepare-0.14/src/packages.dhall
, dependencies = 
    [ "prelude"
    , "effect"
    ]
, sources = [ "src/**/*/.purs" ]
}
```
The libraries that currently compile are tracked here: https://github.com/purescript/package-sets/blob/prepare-0.14/src/updatedLibs.dhall#L8-L137 However, this PR (https://github.com/purescript/package-sets/pull/802) failed to build, so there might be some issues with the `prepare-0.14` branch right now.

Lastly, we're going to make some breaking changes in https://github.com/purescript/purescript-profunctor/issues/23

We make no guarantees about this branch being "stable" as we may change things unexpectedly at any time. So, do so at your own risk.

**Comment by thomashoneyman** [comment]
The PureScript 0.14 package set has been released, so you can use 0.14 packages like any other set.
